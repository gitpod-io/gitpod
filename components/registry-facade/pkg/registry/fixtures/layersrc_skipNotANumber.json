{
    "sourceRef": "localhost:5000/docker-up:skipNaN",
    "content": {
        "abcb15ecdc7e2c6c6ba7d3e80e85b9aa6c44c9c4b14a5625e3d44d700a22d4f7": "eyJjcmVhdGVkIjoiMjAyMC0xMS0wNlQxMDozNDo1My4wMDAxNzUzOFoiLCJhcmNoaXRlY3R1cmUiOiJhbWQ2NCIsIm9zIjoibGludXgiLCJjb25maWciOnsiRW52IjpbIlBBVEg9L3Vzci9sb2NhbC9zYmluOi91c3IvbG9jYWwvYmluOi91c3Ivc2JpbjovdXNyL2Jpbjovc2JpbjovYmluIl0sIldvcmtpbmdEaXIiOiIvdXNyL2JpbiIsIkxhYmVscyI6eyJza2lwLW4ucmVnaXN0cnktZmFjYWRlLmdpdHBvZC5pbyI6Im5vdC1hLW51bWJlciJ9fSwicm9vdGZzIjp7InR5cGUiOiJsYXllcnMiLCJkaWZmX2lkcyI6WyJzaGEyNTY6NTFmYTFlMjU5ZTU4NmRiNjcyZmViNTc0ZWU1ZTMyNTUxMzczNmQ0ZjU1OWJjNTBiZmQ3YTljNDJhYjk5Mzk0NyIsInNoYTI1NjozNzc0ZTg0ZTBhNjlmNTAxMGQwOWRjZGUwOWM4NDA4NzY3NmE3YmRhMzEwNTNlYzIzZDc5NWZkODAyYjNiZTNmIl19LCJoaXN0b3J5IjpbeyJjcmVhdGVkIjoiMjAyMC0xMS0wNlQxMDozNDo1MC4yMzg1MDYxOTNaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBMQUJFTCBza2lwLW4ucmVnaXN0cnktZmFjYWRlLmdpdHBvZC5pbz1ub3QtYS1udW1iZXIiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMC0xMS0wNlQxMDozNDo1MS4xNjc2ODk5MDJaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIFdPUktESVIgL3Vzci9iaW4ifSx7ImNyZWF0ZWQiOiIyMDIwLTExLTA2VDEwOjM0OjUzLjAwMDE3NTM4WiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIG11bHRpOjU3ZjQ5NmJhOWY1ZDJmYzIwN2YwNjc4MDk2ZjlmMGI5MWJjY2M3ZDQ4YWUwZmEyNTEyMDhhMDgyZTI3MDZmMWEgaW4gLi8gIn1dfQ==",
        "fbc668a54cafaeaab277f2b49ac92fca1bf6466bb28b92fbd523477e783c47c9": "eyJzY2hlbWFWZXJzaW9uIjoyLCJjb25maWciOnsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5jb250YWluZXIuaW1hZ2UudjEranNvbiIsImRpZ2VzdCI6InNoYTI1NjphYmNiMTVlY2RjN2UyYzZjNmJhN2QzZTgwZTg1YjlhYTZjNDRjOWM0YjE0YTU2MjVlM2Q0NGQ3MDBhMjJkNGY3Iiwic2l6ZSI6MTc1MX0sImxheWVycyI6W3sibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjpjYzhhNzE4YmEyMGViNDUwNWU2NDg2MWQ2OGZmOTZjMWE1OGU1NmU1MGFlNGVlNGNkYjlhYWUyMDcyODY1Mzc5Iiwic2l6ZSI6MTA5fSx7Im1lZGlhVHlwZSI6ImFwcGxpY2F0aW9uL3ZuZC5kb2NrZXIuaW1hZ2Uucm9vdGZzLmRpZmYudGFyLmd6aXAiLCJkaWdlc3QiOiJzaGEyNTY6ZDU3NDUxYjM1NTMwYzc2MDA4YTUxZjFlOTM2NTE0ZmRmODEyZGFkMDM0NjEyNDIzZmJiY2ZhN2ZhNmM5OWNlOSIsInNpemUiOjUxODc0MDh9XX0=",
        "localhost:5000/docker-up:skipNaN": "eyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmRpc3RyaWJ1dGlvbi5tYW5pZmVzdC52Mitqc29uIiwiZGlnZXN0Ijoic2hhMjU2OmZiYzY2OGE1NGNhZmFlYWFiMjc3ZjJiNDlhYzkyZmNhMWJmNjQ2NmJiMjhiOTJmYmQ1MjM0NzdlNzgzYzQ3YzkiLCJzaXplIjo3MzV9"
    }
}