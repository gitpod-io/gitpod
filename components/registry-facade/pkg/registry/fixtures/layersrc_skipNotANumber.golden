{
    "error": "skipN layer label: strconv.ParseUint: parsing \"not-a-number\": invalid syntax",
    "layer": null,
    "envs": null
}